donkeycar/pipeline/sequence.py

```diff
@@ -1,7 +1,6 @@
-from typing import Any, Callable, Generic, Iterable, Iterator, List, Sized, Tuple, TypeVar
+from typing import Any, Callable, Generic, Iterable, Iterator, List, Sized, \
+    Tuple, TypeVar
```

Comment:
You should be able to completely undo changes to this file now. I fixed the `sequence`s so they behave per the iterator protocol. (https://github.com/autorope/donkeycar/pull/704)